src/utils/fetchQuickstarts.ts

```diff
@@ -59,11 +67,11 @@ async function fetchQuickstarts(
   });
 
   return content.map((item) => {
-    const name = item.metadata.name;
+    const name = item?.metadata?.name;
```

Comment:
I am curious why we need to include optional chaining here now? `content` is of type `ExtendedQuickStart[]`, where `ExtendedQuickStart` extends patternflys `Quickstart` type. As far as I can tell - this type structure does not allow for `undefined`, `null`, etc `item`s in the array, and patternflys `Quickstart` type enforces that `metadata.name` _must_ be defined. Was there an error you were seeing that this resolves?

For reference, here is patternfly's type: https://github.com/patternfly/patternfly-quickstarts/blob/main/packages/module/src/utils/quick-start-types.ts#L9

Reply:
I believe that I was seeing some errors, but it seems to work seamlessly without the optional chaining now.